core/login.go

```diff
@@ -17,15 +17,18 @@ package core
 
 import (
 	"fmt"
-	"net/url"
 	. "github.com/vmware/priam/util"
+	"net/url"
 )
 
+/* Local user domain as defined in IDM (might change in the future when multiple local domains will be supported) . */
+const LOCAL_USER_DOMAIN = "Local Users"
+
 /* ClientCredsGrant takes a clientID and clientSecret and makes a request for an access token.
    The access token is returned in a string prefixed by the token type for use in an http
    authorization header.
 */
-func ClientCredentialsGrant(ctx *HttpContext, path, clientID, clientSecret string) (authHeader string, err error) {
+func ClientCredentialsGrant(ctx *HttpContext, path, clientID, clientSecret string, unused string) (authHeader string, err error) {
```

Comment:
yeah, I see what you mean. I probably would have rearranged the cli.go code to not use the func var so the unused parameter would not be needed here, but it's not worth changing. I'm surprised "unused" doesn't get a compiler error and would expect you'd need to use '_' instead.